test/template-tests/README.md

```diff
@@ -0,0 +1,58 @@
+﻿### Running Tests
+
+Test can be run directly in PowerShell, or run from the command line using a wrapper script.
+
+You can run the full suite of tests by using Test-AzureRMTemplate.cmd (on Windows) or Test-AzureRMTemplate.sh (on Linux), and passing in the path to a template.
+
+This will run the full suite of applicable tests on your template.  To run a specific group of tests, use:
+
+
+    Test-AzureRMTemplate -TemplatePath $thePathToYourTemplate -Test deploymentTemplate 
+    # This will run deployment template tests on all appropriate files
+    
+    Test-AzureRMTemplate -TemplatePath $thePathToYourTemplate -Test "Resources Should Have Location" 
+    # This will run the specific test, 'Resources Should have Location', on all appropriate files
+
+    Test-AzureRMTemplate -TemplatePath $thePathToYourTemplate -Test "Resources Should Have Location" -File MyNestedTemplate.json 
+    # This will run the specific test, 'Resources Should have Location', but only on MyNestedTemplate.json 
+
+
+#### Running Tests on Linux
+
+Before you run the tests on Linux, you'll need to [install PowerShell Core](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-6).
+
+#### Running Tests in PowerShell
+
+To run the tests in PowerShell, you'll need to import the module.
+
+    Import-Module .\AzRMTester.psd1 # assuming you're in the same directory as .\AzRMTester.psd1
+
+You can then test a particular path by using:
+
+    Test-AzureRMTemplate -TemplatePath /yourTemplatePath/

```

Comment:
checking the pathing here...

/foo - means off the root?
foo - means off the current path
../foo - up one folder?

and the trailing / is optional (so just remove it)?